sn_node/src/node.rs

```diff
@@ -368,8 +368,10 @@ impl Node {
                     error!("Failed to remove local record: {e:?}");
                 }
             }
-            NetworkEvent::GossipsubMsgReceived { topic, msg }
-            | NetworkEvent::GossipsubMsgPublished { topic, msg } => {
+            NetworkEvent::GossipsubMsgPublished { topic: _ } => {
+                // Nothing to do here
+            }
```

Comment:
We propagated the `msgs published by self` here, since the `spend test` counts the number of `royalty payments` made and if the `genesis node` validates a payment and publishes a `gossip for that payment`, any incoming gossip msg for that payment is filiterd out by the genesis node and will not reach the `swarm layer`. Hence we propgated any msg that we published to be handled here. 

This was the PR for it https://github.com/maidsafe/safe_network/pull/859

Reply:
Aha, okay I'll revert this.